vanilla-framework/vanilla-framework
# Add sidebar nav

## Done

Add navigation--sidebar pattern

## QA

- Pull code
- Run `./run serve --watch`
- View http://0.0.0.0:8101/vanilla-framework/examples/patterns/navigation/sidebar/
- Check docs make sense

## Details

Fixes https://www.github.com/vanilla-framework/vanilla-docs-theme/issues/45
